OpenInSublimeText/OSTOpenInSublimeText.m

```diff
@@ -112,7 +112,7 @@ - (void) openCurrentFileWithSublimeTextAtLocation:(CursorLocation)location {
         
         NSTask *task = [[NSTask alloc] init];
         
-        task.launchPath = @"/Applications/Sublime Text.app/Contents/SharedSupport/bin/subl";
+        task.launchPath = @"/Applications/Sublime Text 2.app/Contents/MacOS/Sublime Text 2";
```

Comment:
I believe we actually want to leave this how it was. Sublime Text 3 is actually the latest version, which is installed as /Applications/Sublime Text.app by default. So this would be dropping support for Sublime Text 3 and adding support for Sublime Text 2